ts/src/connections/tunnelRelayTunnelHost.ts

```diff
@@ -462,6 +463,7 @@ export class TunnelRelayTunnelHost extends TunnelConnectionSession implements Tu
             config.addService(PortForwardingService);
         });
         session.trace = this.trace;
+        session.onReportProgress(this.raiseReportProgress, this, this.sshSessionDisposables);
```

Comment:
Is this OK if progress is reported from multiple SSH sessions? The main one that connects sdk host to Relay, and different client sessions? The progress sequence may be quite chaotic.

Reply:
I think so. The progress events aren't very verbose. Really only tracks 6 steps to help diagnose connection issues

Reply:
But there will be repetitive and out-of-order steps coming from different sessions.

Reply:
thanks for pointing this case out! As discussed, I'll switch to using event args and add a clientSessionId that can be used to determine which client session is reporting the event.